pkg/driver/node/mounter/pod_unmounter.go

```diff
@@ -227,8 +227,14 @@ func (u *PodUnmounter) writeExitFile(podPath string) error {
 
 // cleanupCredentials removes credentials associated with the Mountpoint Pod
 func (u *PodUnmounter) cleanupCredentials(mpPod *corev1.Pod) error {
+	volumeID := mpPod.Annotations[mppod.AnnotationVolumeId]
```

Comment:
I'd suggest using:

```go
volumeID, exists := mpPod.Annotations[mppod.AnnotationVolumeId]
```

Also, what happens if neither the annotation nor the label for `VolumeID` exists? we'll we fail to clean up creds?